Docs/mesh.md

```diff
@@ -40,15 +44,18 @@ For instance
         m.normals().push_back( {0, 0, 1} );
         m.m_triangles.push_back( {0, 1, 2} );
 
-        auto handle1 = m.attribManager().addAttrib<Vector3>( "vector3_attrib" );
-        m.attribManager().getAttrib( handle1 ).data().reserve( 3 );
-        m.attribManager().getAttrib( handle1 ).data().push_back( {1, 1, 1} );
-        m.attribManager().getAttrib( handle1 ).data().push_back( {2, 2, 2} );
-        m.attribManager().getAttrib( handle1 ).data().push_back( {3, 3, 3} );
+        auto handle1 = m.addAttrib<Vector3>( "vector3_attrib" );
+        m.getAttrib( handle1 ).data().reserve( 3 );
+        m.getAttrib( handle1 ).data().push_back( {1, 1, 1} );
+        m.getAttrib( handle1 ).data().push_back( {2, 2, 2} );
+        m.getAttrib( handle1 ).data().push_back( {3, 3, 3} );
```

Comment:
Can I do this: 
```
auto handle1 = m.addAttrib<Vector3>( "vector3_attrib" );
auto &buf = m.getAttrib( handle1 ).data();
buf.reserve( 3 );
buf.push_back( {1, 1, 1} );
buf.push_back( {2, 2, 2} );
buf.push_back( {3, 3, 3} );
```
The underlying question is: what is the cost of calling `m.getAttrib( handle1 )` many times ?

Reply:
Yes you can ;)

Reply:
Can you add the complexity of these functions in the AttribManager methods doc ?
Also, it could be nice to refer from TriangleMesh to AttribManager for the methods that are delegated to AttribManager, something like:
```c++
    /// Get attribute by handle (const).
    /// \see AttribManager::getAttrib
    template <typename T>
    inline const Attrib<T>& getAttrib( AttribHandle<T> h ) const {
        return m_vertexAttribs.getAttrib( h );
    }
```

Reply:
yes, forgot the getters and copyAttributes